packages/components/calendar/src/use-calendar-picker.ts

```diff
@@ -234,6 +234,7 @@ export const useCalendarPicker = <T extends UseCalendarProps<any>>(
 
   const containerRef = useRef<HTMLDivElement>(null)
   const inputRef = useRef<HTMLInputElement>(null)
+  const calendarId = useId()
```

Comment:
@cidkumagai 

It's okay to call useId() internally to generate an ID only when id is undefined.

```ts
 id ??= useId()
```

Reply:
or 

```ts
const calendarId = id ?? useId()
```